### DISCUSSION TITLE
Hamclock tile

### DISCUSSION THREAD

**Discussion by OK1SLM** [VA3HDL/hamdashboard#38] [Q&A]
How do I edit the cfg so that the Hamclock tile updates automatically?

**Comment by VA3HDL** [comment]
Hi,

On the latest version of the dashboard you can have a live website embedded on the tiles.
You can use a link like this to achieve the result:

"iframe|http://192.168.0.111:8081/live.html",

Replace the IP address above with your Hamclock IP address.

![Screenshot from 2024-11-17 08-26-16](https://github.com/user-attachments/assets/a44700ca-131d-4034-8c26-817b2ef528db)
